src/fragments/lib-v1/auth/native_common/getting_started/common.mdx

```diff
@@ -36,15 +36,15 @@ import flutter8 from '/src/fragments/lib-v1/auth/flutter/getting_started/20_inst
 
 The most common way to use Authentication with Amplify is via the Amplify CLI, which allows you to create new Amazon Cognito resources or import existing ones. However, you can also use the Amplify Studio console to configure authentication or use the `Amplify.configure()` method to set up authentication with existing resources.
 
-import ios50 from '/src/fragments/lib/auth/ios/getting_started/50_configureBackend.mdx';
+import ios50 from '/src/fragments/lib-v1/auth/ios/getting_started/50_configureBackend.mdx';
```

Comment:
These 3 fixes from lib to lib-v1 seem to already exist in main. Lets only fix the ones that we introduced in the PR. This file doesn't look like it was touched in the current PR until now.